.babelrc

```diff
@@ -1,5 +1,12 @@
 {
   "stage": 1,
-  "loose": ["es6.modules", "es6.classes"],
-  "sourceMaps" : "inline"
+  "loose": true,
+  "sourceMaps": "inline",
+  "optional"  : [
+    "minification.propertyLiterals",
+    "minification.memberExpressionLiterals",
+    "utility.inlineEnvironmentVariables",
```

Comment:
This means it will automatically replace things like `process.env.NODE_ENV` with `"development"`